CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+### Before you Start
+- Unless stated in README.md. Make sure you are pulling the source code from main branch with recursive variable to pull all submodules Collapse need to run.
+
+    ```git pull https://github.com/neon-nyan/Collapse --recursive```

```

Comment:
To ensure parity with the line above, change the command to use `--recurse` instead of `--recursive`. They are both aliases so it shouldn't matter, but better to keep it consistent.

Reply:
Applied